kong/pdk/service/request.lua

```diff
@@ -141,7 +142,8 @@ local function new(self)
       error("path must be a string", 2)
     end
 
-    if string_sub(path, 1, 1) ~= "/" then
+    -- 47 = 0x2f = '/'
+    if string_byte(path) ~= 47 then
```

Comment:
We can do something similar to https://github.com/Kong/kong/blob/master/kong/tools/uri.lua#L42 to avoid magic numbers in the code.

Reply:
add a local SLASH in function new()